src/controllers/csi/gc/images.go

```diff
@@ -0,0 +1,87 @@
+package csigc
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	_os "github.com/Dynatrace/dynatrace-operator/src/controllers/csi/gc/os"
```

Comment:
I don't like it starting with underscore but I have no better idea. `dt_os` maybe?
How about moving it to more general level, like ~~`dynatrace-operator/src/utils`~~ `dynatrace-operator/src/os`?